Makefile.toml

```diff
@@ -261,21 +261,10 @@ cargo install \
 '''
 ]
 
-[tasks.build-packages]
-dependencies = ["build-tools", "publish-setup"]
-script = [
-'''
-export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
-cargo build \
-  ${CARGO_BUILD_ARGS} \
-  ${CARGO_MAKE_CARGO_ARGS} \
-  --manifest-path packages/Cargo.toml \
-  --all
-'''
-]
-
 [tasks.build-kmod-kit]
-dependencies = ["build-packages"]
+# FIXME - oops, it's weird to build a variant just to get the kernel package built
+# FIXME - maybe a variant's kernel package should become a first-class concept so it could be built here
+dependencies = ["build-variant"]
```

Comment:
I think what I want to do here is create a `build-kernel` target and call it here instead of `build-packages` or `build-variant`.